GNUmakefile

```diff
@@ -264,7 +264,7 @@ test-nomad: dev ## Run Nomad test suites
 	$(if $(ENABLE_RACE),GORACE="strip_path_prefix=$(GOPATH)/src") go test \
 		$(if $(ENABLE_RACE),-race) $(if $(VERBOSE),-v) \
 		-cover \
-		-timeout=900s \
+		-timeout=30m \
```

Comment:
30 minutes is too long.. could we merge this but then watch travis runtimes for a week and adjust downwards?